modules/distribution/resources/operation_policies/specifications/ccRewriteResourcePath_v1.json

```diff
@@ -0,0 +1,36 @@
+{
+  "category": "Mediation",
+  "name": "rewriteResourcePath",
+  "version": "v1",
+  "displayName": "Rewrite Resource Path",
+  "description": "This policy allows you to rewrite the resource path of a request",
+  "policyAttributes": [
+    {
+      "name": "newResourcePath",
+      "displayName": "New Resource Path",
+      "description": "Name of the new resource path",
+      "validationRegex": "^\\/?(({uri\\.var\\.[\\w\\/\\-]+})|[\\w\\/\\-\\s])+$",
+      "type": "String",
+      "required": true
+    },
+    {
+      "name": "includeQueryParams",
+      "displayName": "Include Query Parameters?",
```

Comment:
Can we  change  this ? ( `Include Query Parameters?` -> `Include Query Parameters` )

Reply:
fixed with https://github.com/wso2/product-apim/pull/12669/commits/c55115b52842f6c0349af3be192ac5901d91405f